JS/Lesson15/saffet/script.js

```diff
@@ -67,113 +67,348 @@ switch (day) {
 */
 
 // 1. Write a program that takes in a number and checks if it's even or odd using if else statement.
-function isEvenOrOdd(number) {
-	if (number % 2 === 0) {
-		console.log('it is even');
-	} else {
-		console.log('it is odd');
+console.log("Ex1:");
+function oddOrEven(number){
+	if(number % 2 == 0){
+		console.log("Number is even :)");
+	}
+	else{
+		console.log("Number is odd :(");
 	}
 }
 
-isEvenOrOdd(4);
-isEvenOrOdd(7);
-
-// 2. Write a program that takes age as input. If the age is less than 18, print "Sorry, you are not old enough to vote". If the age is between 18 and 65 (inclusive), print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
-
-function checkAge(age) {
-	if (age < 18) {
-		console.log('Sorry, you are not old enough to vote');
-	} else if (age >= 18 && age <= 65) {
-		console.log('You are eligible to vote');
-	} else {
-		console.log('you are a citizen');
+oddOrEven(156);
+oddOrEven(333);
+console.log("-------------------------------------------------");
+// 2. Write a program that takes age as input. If the age is less than 18, print "Sorry, you are not old enough to vote". 
+// 	  If the age is between 18 and 65 (inclusive), print "You are eligible to vote". If the age is greater than 65, print 
+// 	  "You are a senior citizen".
+console.log("Ex2:");
+function checkAge(age){
+	if(age < 18){
+		console.log("Sorry, you are not old enough to vote");
+	}
+	else if(age>18 && age<65){
```

Comment:
In this case, checkAge does not take into account numbers 18 and 65 (they both will return "You are a senior citizen") which is not expected. Include both numbers in the condition like this: `(age >= 18 && age <= 65)`